package/src/core/types.ts

```diff
@@ -142,3 +142,8 @@ export interface ExtendedHooks<TPlugins extends Array<AnyPlugin>> {
 		AddedParam<TPlugins, "astro:build:done">
 	>;
 }
+
+export interface DevToolbarFrameworkAppProps {
+	canvas: ShadowRoot;
+	renderWindow: HTMLElement;
+}
```

Comment:
```suggestion
export interface DevToolbarFrameworkAppProps {
	canvas: Parameters<import("astro").DevToolbarApp["init"]>[0];
	eventTarget: Parameters<import("astro").DevToolbarApp["init"]>[1];
	// Make sure to import a type from astro at the top so that this global get augmented
	renderWindow: HTMLElementTagNameMap['astro-dev-toolbar-window'];
}
```